modules/preferred-ibm-z-system-requirements.adoc

```diff
@@ -2,6 +2,21 @@
 //
 // * installing/installing_ibm_z/installing-ibm-z.adoc
 // * installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc
+// * installing/installing_ibm_z/installing-ibm-z-lpar.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z-lpar.adoc
+
+ifeval::["{context}" == "installing-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-ibm-z-lpar"]
+:ibm-z-lpar:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z-lpar"]
+:ibm-z-lpar:
+endif::[]
```

Comment:
You set `ibm-z-lpar` here and on line 14, but never use it. Does it make sense to replace instances of `ifndef::ibm-z[]` with `ifdef::ibm-z-lpar`?